README.md

```diff
@@ -65,6 +69,9 @@ SharpPcap usage.
 
 The examples are also a great resource a they show working examples using the latest API.
 
+* NativeLibrary is used for improved capture library resolution
+  * Improves library reosolution situation on Linux distros where there is a libpcap.so.X.Y symlink but no libpcap.so symlink
```

Comment:
I still have not done this yet in #236, I just made it a lot more easier to implement in a future PR